app.js

```diff
@@ -0,0 +1,45 @@
+let Books = JSON.parse(localStorage.getItem('Books' || '[]'));
+if (Books === null) {
+  Books = [];
+}
+
+function removeBook(title) {
+  const book = document.getElementById(title);
+  book.remove();
+  Books = Books.filter((bookObj) => bookObj.title !== title);
+  localStorage.setItem('Books', JSON.stringify(Books));
+}
+
+function addBook(bookObj) {
+  const bookList = document.getElementById('book-list');
+  const book = document.createElement('li');
+  book.setAttribute('id', bookObj.title);
+  book.innerHTML = `
+      <h3> ${bookObj.title} </h3> 
+      <p>  ${bookObj.author} </p>
+      `;
+  const removeBtn = document.createElement('button');
+  removeBtn.innerHTML = 'Remove';
+  removeBtn.addEventListener('click', () => removeBook(bookObj.title));
+  book.appendChild(removeBtn);
+  bookList.appendChild(book);
+}
+
+Books.forEach((book) => {
+  addBook(book);
+});
+
+const form = document.querySelector('#form');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const book = {};
+  book.title = document.getElementById('book-title').value;
+  book.author = document.getElementById('book-author').value;
+
+  if ((book.title && book.author) !== '') {
+    Books.push(book);
+    localStorage.setItem('Books', JSON.stringify(Books));
+    addBook(book);
```

Comment:
- [ ] As per this requirement below, Please add the functionality of adding a new book to a book collection inside a separate function and you can call it here inside the submit event listener.

![image](https://user-images.githubusercontent.com/68796232/234137252-6e059c9f-89f4-444a-9eed-1ae506500845.png)

Reply:
Great advice, Thank you. 
It's done.